application/src/main/java/org/thingsboard/mqtt/broker/actors/client/ClientActor.java

```diff
@@ -185,6 +188,14 @@ protected boolean doProcess(TbActorMsg msg) {
                             }
                         }
                         break;
+
+                    case WRITABLE_CHANNEL_MSG:
+                        backpressureManager.onChannelWritable(state);
+                        break;
+                    case NON_WRITABLE_CHANNEL_MSG:
+                        backpressureManager.onChannelNonWritable(state.getClientId(), state.getCurrentSessionCtx());
```

Comment:
I noticed that this method is used only in one place. Consider to update to:

```suggestion
                        backpressureManager.onChannelNonWritable(state);
```